src/NHibernate.Test/CfgTest/LoadVirtualAssembyTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using NHibernate.Cfg;
+using System.CodeDom.Compiler;
+using Microsoft.CSharp;
+using System.Reflection;
+using NHibernate.Mapping.ByCode;
+using NHibernate.Mapping.ByCode.Conformist;
+
+namespace NHibernate.Test.CfgTest
+{
+	[TestFixture]
+	class LoadVirtualAssembyTest
```

Comment:
More tests should be added for covering other methods of `ReflectHelperMemoryAssembly`, like `RemoveAssembly`, `GetType(string)`, ... Or those methods should be removed if they have no usage (seems to the case).